hyperon_das/das.py

```diff
@@ -499,17 +514,20 @@ def get_traversal_cursor(self, handle: str, **kwargs) -> TraverseEngine:
         except AtomDoesNotExist:
             raise GetTraversalCursorException(message="Cannot start Traversal. Atom does not exist")
 
-    @staticmethod
-    def about() -> dict:
-        return {
-            'das': {
-                'name': 'hyperon-das',
-                'version': get_package_version('hyperon_das'),
-                'summary': 'Query Engine API for Distributed AtomSpace',
-            },
-            'atom_db': {
-                'name': 'hyperon-das-atomdb',
-                'version': get_package_version('hyperon_das_atomdb'),
-                'summary': 'Persistence layer for Distributed AtomSpace',
-            },
-        }
+    def create_field_index(self, atom_type: str, field: str, type: str = None) -> str:
+        """Create an index for a field for all Atoms of the specified type
+
+        Args:
+            atom_type (str): Type of the Atom. Could be 'link' or 'node'
+            field (str): field where the index will be created
+            type (str, optional): The type of the Link/Node. Defaults to None.
```

Comment:
The explanation for this parameter says nothing useful.

Suggestion: "Only atoms of the passed type will be indexed"